CMakeLists.txt

```diff
@@ -89,9 +89,11 @@ IF ( NOT VERSION )
    message( STATUS "version base: ${VERSION-BASE}" )
    message( STATUS "version remainder: ${VERSION-REMAINDER}" )
    SET(BUILD_NUMBER ${VERSION-BASE})
-   SET(VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_NUMBER}-${VERSION-REMAINDER})
+   SET(SOFT_VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_NUMBER}-${VERSION-REMAINDER})
```

Comment:
Did you mean `soversion`? AFAIK `SOFT_VERSION` is not a reserved keyword in cmake
https://cmake.org/cmake/help/latest/prop_tgt/SOVERSION.html#prop_tgt:SOVERSION